addons/tagging/functions/fnc_compileConfigTags.sqf

```diff
@@ -46,8 +46,6 @@
 
     if (!_failure) then {
         GVAR(cachedTags) pushBack [_class, _displayName, _requiredItem, _textures, _icon];
-        if !(_requiredItem in GVAR(cachedRequiredItems)) then {
-            GVAR(cachedRequiredItems) pushBack _requiredItem;
-        };
+        GVAR(cachedRequiredItems) pushBackUnique _requiredItem;
     };
-} forEach ("true" configClasses (configFile >> "ACE_Tags"));
+} forEach (configProperties [configFile >> "ACE_Tags"]);
```

Comment:
Will also include non-class entries, which would error out if you do e.g. `_x >> "displayName"`
I guess that's fine though as the class is not supposed to have such entries.